papis/document.py

```diff
@@ -421,7 +421,7 @@ def describe(document: Union[Document, Dict[str, Any]]) -> str:
     """
     return papis.format.format(
         papis.config.getstring("document-description-format"),
-        document)
+        document, default="")
```

Comment:
```suggestion
        document, default=doc.get("papis_id", ""))
```
? The `papis_id` key should at least be in all the documents saved to disk..